root-files/opt/flownative/lib/nginx-legacy.sh

```diff
@@ -193,9 +200,9 @@ EOM
 EOM
     elif [ -n "${BEACH_PERSISTENT_RESOURCES_FALLBACK_BASE_URI}" ]; then
         cat >>"${NGINX_CONF_PATH}/sites-enabled/site.conf" <<-EOM
-    location ~* ^/_Resources/Persistent/(.*)$ {
+    location ~* "^${BEACH_PERSISTENT_RESOURCES_BASE_PATH}(.*)$ {
```

Comment:
Is that always the same value as before?

Reply:
If someone set the `BEACH_PERSISTENT_RESOURCES_BASE_PATH` to something else for a fallback-setup, this wouldn't have had any effect. So I guess we can dare to apply this change.